http/Cargo.toml

```diff
@@ -18,6 +18,7 @@ version = "0.7.2"
 hyper = { default-features = false, features = ["client", "http1", "http2", "runtime"], version = "0.14" }
 hyper-rustls = { default-features = false, optional = true, features = ["http1", "http2"], version = "0.23" }
 hyper-tls = { default-features = false, optional = true, version = "0.5" }
+hyper-trust-dns = { default-features = false, optional = true, version = "0.3.1" }
```

Comment:
This should be moved to the optional dependencies section, i guess this whole file needs to be reworked at some point later. it's fine for now